README.md

```diff
@@ -119,301 +156,238 @@ docker-compose up
 
 Configuration
 -------------
-
-Change your default database connection name in `config/database.php`:
-
-```php
-'default' => env('DB_CONNECTION', 'mongodb'),
-```
-
-And add a new mongodb connection:
+You can use MongoDB either as a main database, either as a side database. To do so, add a new `mongodb` connection to `config/database.php`:
 
 ```php
 'mongodb' => [
-    'driver'   => 'mongodb',
-    'host'     => env('DB_HOST', 'localhost'),
-    'port'     => env('DB_PORT', 27017),
-    'database' => env('DB_DATABASE'),
-    'username' => env('DB_USERNAME'),
-    'password' => env('DB_PASSWORD'),
-    'options'  => [
-        'database' => 'admin' // sets the authentication database required by mongo 3
-    ]
+    'driver' => 'mongodb',
+    'host' => env('DB_HOST', '127.0.0.1'),
+    'port' => env('DB_PORT', 27017),
+    'database' => env('DB_DATABASE', 'homestead'),
+    'username' => env('DB_USERNAME', 'homestead'),
+    'password' => env('DB_PASSWORD', 'secret'),
+    'options' => [
+        'database' => 'admin', // required with Mongo 3+
+
+        // here you can pass more settings
+        // see https://www.php.net/manual/en/mongoclient.construct.php under "Parameters" for a list of complete parameters you can use
```

Comment:
Link on deprecated extension

Reply:
Actual link [here](https://www.php.net/manual/en/mongodb-driver-manager.construct.php)

Reply:
Can you change it to https://www.php.net/manual/en/mongodb-driver-manager.construct.php ?